src/app/components/Grid/index.jsx

```diff
@@ -269,6 +270,7 @@ const PopOutAtGroup5 = styled(GridItemMedium)`
     @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
       max-height: 0; /* [1] */
       padding-top: 0.25rem;
+      padding-left: ${GEL_SPACING_HLF_TRPL};
```

Comment:
I think there's a general favourability towards `margin` over `padding`, and I think we'd need to add space to the right of the timestamp for right-to-left services. Perhaps something like:
```
margin-left: ${GEL_SPACING_HLF_TRPL};
margin-right: ${GEL_SPACING_HLF_TRPL};
```
Would add space for both ltr and rtl services, and not use different css for either.

Reply:
Thanks for the review @HarryVerhoef 
 I have addressed both concerns in this PR https://github.com/bbc/simorgh/pull/9190 because of GPG signing issues. Please take a look when you get some time